### DISCUSSION TITLE
Can I suppress ANN101 for the first arguments ('cls', 'self') of methods?

### DISCUSSION THREAD

**Discussion by vadim-su** [astral-sh/ruff#4542] [Q&A] [answered]
```py
class TestClass:
    @classmethod
    def print(cls, string: str):  # <-- Missing type annotation for `cls` in method Ruff(ANN101)
        print(string)
```

Is it possible?

**Comment by charliermarsh** [comment, marked as answer]
Yup! You can disable the `ANN101` and `ANN102` rule to turn these off. They're dedicated rules for that reason.

**Reply by vadim-su** [reply]
![image](https://github.com/charliermarsh/ruff/assets/1702003/1a494eb1-5a2b-4516-b0ee-f005bfd0ebac)

I can't find any information about `ANN100` in the docs and TOML parser said 
```"ANN100" is not one of ["A","A0","A00","A001","A002...```
Is it ok?

**Reply by charliermarsh** [reply]
My mistake, it's `ANN101` and `ANN102`.

(I edited my answer to reflect this.)
